support/client/lib/vwf/view/blockly.js

```diff
@@ -232,45 +232,30 @@ define( [ "module", "vwf/view", "jquery", "vwf/model/blockly/JS-Interpreter/acor
                         // If the new node is the same as the old - exit early to prevent 
                         // breaking synchronization.
 
-                        if ( previousActiveNode === newActiveNodeId ) {
+                        if ( previousActiveNode === blocklyNode ) {
                             break;
                         }
 
-                        if ( blocklyNode !== undefined ) {
```

Comment:
Looks like it's all replicating. I'm still seeing scene undefined errors, but those seem to have no effect.